### ISSUE TITLE
Beta, Streaming Booste 80m = sound bug

### ISSUE THREAD

**Issue by Amerigowarlock** [sammilucia/cyberpunk-ultra-plus#19]
Streaming boost seems to impact sound, 
set to max causes massive sound issues such as infinite sound loop, crackling etc

and i'm pretty sure there's something happening at 40 as well

Going to leave it at 20 for now as i don't notice any bug there

**Comment by semool** [comment]
Yes i have that to sometimes. Often when i leaving Megabuilding 10. On the Street the Sound is very loud and confuse.
I have The boost at 30 and with this there are no problems.

**Comment by sammilucia** [comment]
hmm ty. I've heard this a few times but it doesn't seem to happen for most people 🤔

**Comment by TteokbokkiNari** [comment]
Have noticed this as well on my end when streaming is set to 80 meters.
